src/tools/compiletest/src/main.rs

```diff
@@ -1043,3 +1076,54 @@ fn test_extract_gdb_version() {
         7012050: "GNU gdb (GDB) 7.12.50.20161027-git",
     }
 }
+
+
+fn check_clang_based_tests_possible(config: &Config) -> Result<(), String> {
```

Comment:
I wonder if we could perhaps simplify this and just set it in rustbuild configuration? It seems like we'll really rarely have a compatible clang, so in general we'll want to only set this option on one builder to pass in the clang used with lldb and enable the tests

Reply:
You mean, don't do a check at all?

Reply:
Yeah I think we can skip checking this entirely because in theory it's only configured on one builder and on maybe one or two dev's computers which in theory don't need too many checks here.